core-java-modules/core-java-lang-5/src/main/java/com/baeldung/generics/AnimalDemo.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.generics;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class AnimalDemo {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AnimalDemo.class);
```

Comment:
Although this variable has the modifier `static final`, it is not a constant and should not appear in uppercase. In this case, we have a final reference so it should be in lowercase.